tests/queries/0_stateless/03405_ssd_cache_layout.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+# Tags: no-replicated-database
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
```

Comment:
@ahmadov .sql tests are simpler than .sh tests (fewer degrees of freedom, less complexity), and therefore preferred. In some cases .sh tests are needed but as far as I see, this isn't one of them.

It is possible to do "negative" tests in .sql tests as well - see tests/queries/0_stateless/02354_vector_search_index_creation_negative.sql for an example. One can only check the error code instead the full error string but that is fine.

Therefore, could you please try to convert this test to .sql? Thanks!

Reply:
yes, the only reason I used `.sh` tests to check the exact error message but as you mentioned it's not so important since the error codes can be used as well. Moved them into `.sql` tests with 1d06a0c061cc1822e6e1a57db4e36a3f938428ad